elichika/tests/syntax/For.py

```diff
@@ -141,6 +147,14 @@ def main():
                            [np.random.rand(4, 3).astype(np.float32), 2, 5],
                            subname='double_for', backprop=True)
 
+    # Bugs.
+    model = lambda: B()
+    testtools.generate_testcase(model, args, subname='lambda_bug')
```

Comment:
What is the expected behavior here?  For me,
```
$ build/tools/dump out/elichika_syntax_For_closure_bug > a
$ build/tools/dump out/elichika_syntax_For_lambda_bug > b
$ diff -u a b
```
shows
```
--- a	2019-04-29 17:05:30.658437913 +0900
+++ b	2019-04-29 17:05:38.354337738 +0900
@@ -1,4 +1,4 @@
-=== out/elichika_syntax_For_closure_bug ===
+=== out/elichika_syntax_For_lambda_bug ===
 ir_version: 5
 producer_name: "elichika"
 producer_version: "0.1"
@@ -262,7 +262,7 @@
 opset_import {
   version: 10
 }
-=== out/elichika_syntax_For_closure_bug/test_data_set_0/input_0_0.pb ===
+=== out/elichika_syntax_For_lambda_bug/test_data_set_0/input_0_0.pb ===
 dims: 4
 dims: 5
 data_type: 1
@@ -287,7 +287,7 @@
 float_data: 0.130388752
 float_data: 0.653830528
 name: "in_0_L.-1"
-=== out/elichika_syntax_For_closure_bug/test_data_set_0/input_0_1.pb ===
+=== out/elichika_syntax_For_lambda_bug/test_data_set_0/input_0_1.pb ===
 dims: 4
 dims: 5
 data_type: 1
@@ -312,7 +312,7 @@
 float_data: 0.46937564
 float_data: 0.190825194
 name: "in_0_L.-1"
-=== out/elichika_syntax_For_closure_bug/test_data_set_0/input_0_2.pb ===
+=== out/elichika_syntax_For_lambda_bug/test_data_set_0/input_0_2.pb ===
 dims: 4
 dims: 5
 data_type: 1
@@ -337,7 +337,7 @@
 float_data: 0.648225069
 float_data: 0.292232305
 name: "in_0_L.-1"
-=== out/elichika_syntax_For_closure_bug/test_data_set_0/input_0_3.pb ===
+=== out/elichika_syntax_For_lambda_bug/test_data_set_0/input_0_3.pb ===
 dims: 4
 dims: 5
 data_type: 1
@@ -362,11 +362,11 @@
 float_data: 0.254202753
 float_data: 0.447350442
 name: "in_0_L.-1"
-=== out/elichika_syntax_For_closure_bug/test_data_set_0/input_1.pb ===
+=== out/elichika_syntax_For_lambda_bug/test_data_set_0/input_1.pb ===
 data_type: 7
 int64_data: 4
 name: "in_1_L.-1"
-=== out/elichika_syntax_For_closure_bug/test_data_set_0/output_0.pb ===
+=== out/elichika_syntax_For_lambda_bug/test_data_set_0/output_0.pb ===
 dims: 4
 dims: 5
 data_type: 1
```
which is reasonable. Your comment git checkout -b Rishav1-mylstm-elichika mylstm-elichika
git pull git://github.com/Rishav1/chainer-compiler.git mylstm-elichika suggests the diff is larger than mine for you?

Reply:
My bad. This lambda_bug test has been moved to Linear.py. This test doesn't show any error and works as expected. Plus the B2 test was commented out by mistake. Fixed it in a consequent commit.